Resources/Prototypes/Entities/Structures/Windows/plastitanium.yml

```
@@ -167,6 +170,9 @@
   name: plastitanium window
   parent: PlastitaniumWindowDiagonalBase
   suffix: diagonal, indestructible
+  components:
+    - type: ExaminableDamage
+      messages: null
```

Gah forgot to save but same thing here:
```suggestion
  components:
  - type: ExaminableDamage
    messages: null
```

---

Read other comment (: